.env

```diff
@@ -1,4 +1,4 @@
-REACT_APP_BASE_API_URL=https://baris.develaws.zenml.io/api/v1
+REACT_APP_BASE_API_URL=https://michael.develaws.zenml.io/api/v1
```

Comment:
we shouldnt change this right?

Reply:
Yes, I think we should go with the michael’s one.

Reply:
@htahir1 Should I merge it?